plugin-locationlayer/src/main/res/values/attrs.xml

```diff
@@ -34,6 +34,10 @@
     <attr name="mapbox_maxZoom" format="float"/>
     <attr name="mapbox_minZoom" format="float"/>
 
+    <!-- Icon scale based on map zoom levels -->
+    <attr name="mapbox_maxZoomIconScale" format="float"/>
```

Comment:
These also need to be publically exposed in the [`public_attrs.xml`](https://github.com/mapbox/mapbox-plugins-android/blob/master/plugin-locationlayer/src/main/res-public/values/public_attrs.xml) file.